Ошибка: HTTPConnectionPool(host='ytimg.com', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002C392A2A320>: Failed to resolve 'ytimg.com' ([Errno 11001] getaddrinfo failed)"))